model/src/test/kotlin/config/OrtConfigurationTest.kt

```diff
@@ -62,6 +62,7 @@ class OrtConfigurationTest : WordSpec({
             }
 
             ortConfig.downloader shouldNotBeNull {
+                archivedLicenseCategories shouldBe listOf("category-a", "category-b")
```

Comment:
`archivedLicenseCategories should containExactly("category-a", "category-b")` creates a nicer error message.